Src/FluentAssertions.Net40/Xml/XmlNodeAssertions.cs

```diff
@@ -0,0 +1,87 @@
+    {
+        public XmlNodeAssertions(XmlNode xmlNode)
+        {
+            Subject = xmlNode;
+        }
+
+        /// <summary>
+        /// Asserts that the current <see cref="XmlNode"/> is equivalent to the <paramref name="expected"/> element.
+        /// </summary>
+        /// <param name="expected">The expected element</param>
+        public AndConstraint<XmlNodeAssertions> BeEquivalentTo(XmlNode expected)
+        {
+            return BeEquivalentTo(expected, string.Empty);
+        }
+
+        /// <summary>
+        /// Asserts that the current <see cref="XmlNode"/> is equivalent to the <paramref name="expected"/> node.
+        /// </summary>
+        /// <param name="expected">The expected node</param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion 
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="reasonArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        public AndConstraint<XmlNodeAssertions> BeEquivalentTo(XmlNode expected, string because, params object[] reasonArgs)
+        {
+            new XmlReaderValidator(new XmlNodeReader(Subject), new XmlNodeReader(expected), because, reasonArgs).Validate(true);
+
+            return new AndConstraint<XmlNodeAssertions>(this);
+        }
+
+        /// <summary>
+        /// Asserts that the current <see cref="XmlNode"/> is not equivalent to
+        /// the <paramref name="unexpected"/> node.
+        /// </summary>
+        /// <param name="unexpected">The unexpected node</param>
+        public AndConstraint<XmlNodeAssertions> NotBeEquivalentTo(XmlNode unexpected)
+        {
+            return NotBeEquivalentTo(unexpected, string.Empty);
+        }
+
+        /// <summary>
+        /// Asserts that the current <see cref="XmlNode"/> is not equivalent to
+        /// the <paramref name="unexpected"/> node.
+        /// </summary>
+        /// <param name="unexpected">The unexpected node</param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion 
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="reasonArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        /// <returns></returns>
+        public AndConstraint<XmlNodeAssertions> NotBeEquivalentTo(XmlNode unexpected, string because, params object[] reasonArgs)
+        {
+            new XmlReaderValidator(new XmlNodeReader(Subject), new XmlNodeReader(unexpected), because, reasonArgs).Validate(false);
```

Comment:
Not really, they are just in-memory and not backed by any unmanaged resource, but I'll add a using to it.